### DISCUSSION TITLE
Offline mdn content and auto revalidate: unneeded CPU usage

### DISCUSSION THREAD

**Discussion by bansan85** [mdn/yari#10304] [Q&A]
I want to run the mdn content offline. When I run `yarn start` and open `http://localhost:5042/en-us/docs/learn/getting_started_with_the_web`, the page is always refreshed every 500ms :

```
Listening on port 5042
Warning! You have not set an EDITOR environment variable. Using the "Open in your editor" button will probably fail.
buildDocumentFromURL(/en-us/docs/learn/getting_started_with_the_web): 689.35ms
buildDocumentFromURL(/en-us/docs/learn/getting_started_with_the_web): 320.873ms
buildDocumentFromURL(/en-us/docs/learn/getting_started_with_the_web): 269.595ms
buildDocumentFromURL(/en-us/docs/learn/getting_started_with_the_web): 463.804ms
buildDocumentFromURL(/en-us/docs/learn/getting_started_with_the_web): 285.044ms
buildDocumentFromURL(/en-us/docs/learn/getting_started_with_the_web): 224.397ms
```

How can I disable this feature ? At the current time, I disabled JavaScript in Firefox but I don't like this solution. It looks to be related to swr and [revalidation](https://swr.vercel.app/docs/revalidation).

I tried to add `REACT_APP_NO_WATCHER=true REACT_APP_DEV_MODE=false REACT_APP_WRITER_MODE=false` when calling `yari-server` but it doesn't help.

**Comment by Bashamega** [comment]
Hello @bansan85 
It seems that this is the way Yari handles hot reloads. If you are just using it to view the docs without modifying the content, you should run it in production mode.
You can run it in production mode by running:
```
NO_WATCH=true NODE_ENV=production yarn start
```
Your way of doing it doesn't affect the watchers.
